src/functions.tsx

```diff
@@ -0,0 +1,78 @@
+import users from './api/users';
+import { TodoWithUser } from './components/types/TodoWithUser';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+import { TodoError } from './components/types/TodoError';
+
+export function getUserById(userId: number): User {
+  return [...users].filter(currentUser => currentUser.id === userId)[0];
```

Comment:
better to use the find method, 'cause you want to find the user, not filter the array